src/solver.f90

```diff
@@ -272,8 +272,8 @@ subroutine divergence_v2p(self, div_u, u, v, w)
       class(field_t), intent(in) :: u, v, w
 
       class(field_t), pointer :: du_x, dv_x, dw_x, &
-                                 u_y, v_y, w_y, du_y, dv_y, dw_y, &
-                                 u_z, w_z, dw_z
```

Comment:
This is actually everywhere but had to comment on a specific example. I would expect the continuation line to start aligned with the first variable like it used to be.